### PR TITLE
Dead code removal, redundant text macros, drying rack now applies TRAIT_FOOD_CHEF_MADE.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
I've been coding a PR these couple days, and I've noticed a few oversights, all related to food, while working on it.

This PR removes an unused, 12 years old datum, and some redundant text macros in a text string, for processable items, that already uses the bold spans. It also makes it so food made with a drying rack gets the chef made trait, similarly to other methods (though not as foolproof).

## Why It's Good For The Game
A few little improvements.

## Changelog

:cl:
fix: food made with drying racks now counts as "chef-made" (ergo, can provide a mild buff).
/:cl:

### CODE REVIEW

**Comment by Dawnseer** [comment]
Finally my tiziran blood sausages and prime beef jerky counts as chef made.

File: code/datums/elements/dryable.dm [hunk 1]
```diff
@@ -20,32 +20,37 @@
...
 	else
 		var/atom/movable/resulting_atom = new dry_result(source.drop_location())
-		ADD_TRAIT(resulting_atom, TRAIT_DRIED, ELEMENT_TRAIT(type))
+		apply_dried_status(resulting_atom, drying_user)
 		qdel(source)
 
+/datum/element/dryable/proc/apply_dried_status(atom/target, datum/weakref/drying_user)
+	ADD_TRAIT(target, TRAIT_DRIED, ELEMENT_TRAIT(type))
+	var/datum/mind/user_mind = drying_user?.resolve()
+	if(drying_user && istype(target, /obj/item/food))
```
Comment (JohnFulpWillard) [inline code/datums/elements/dryable.dm:55]: I don't like type checking for food item, food is a component so we should be using that instead, no?

Comment (Ghommie) [reply code/datums/elements/dryable.dm:55]: Eedible is the component, while food is the item. Beside, the chef made trait is only useful for food items, which have the recipe complexity variable that dictates the duration and strength of the effects of eaten food (so far only an imperceptible speed boost, and shock immunity for high tier stuff).

TL;DR, the trait is only relevant for food items so far.

Comment (Ghommie) [reply code/datums/elements/dryable.dm:55]: However, I believe an `isfood()` macro should be made. It's beyond the scope of this PR.

Comment (JohnFulpWillard) [reply code/datums/elements/dryable.dm:55]: That's dumb it should've been on the component why are they killing me

Comment (Ghommie) [reply code/datums/elements/dryable.dm:55]: It's The Foodening, bro.
